src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -332,20 +332,18 @@ public void TestAddingRelationshipWithDisabledGraphQL()
             // Exception should be thrown as we cannot use an entity (with graphQL disabled) in a relationship.
             DataApiBuilderException ex = Assert.ThrowsException<DataApiBuilderException>(() =>
                 configValidator.ValidateRelationshipsInConfig(runtimeConfig, _metadataProviderFactory.Object));
-            Assert.AreEqual($"entity: {sampleRelationship.TargetEntity} is disabled for GraphQL.", ex.Message);
+            Assert.AreEqual($"Entity: {sampleRelationship.TargetEntity} is disabled for GraphQL.", ex.Message);
             Assert.AreEqual(HttpStatusCode.ServiceUnavailable, ex.StatusCode);
         }
 
         /// <summary>
         /// Test method to check that an exception is thrown when LinkingObject was provided
-        /// while either LinkingSourceField or SourceField is null, and either targetFields or LinkingTargetField is null.
-        /// And the relationship is not defined in the database.
+        /// while either LinkingSourceField or SourceField is null, and the corresponding targetFields
+        /// or LinkingTargetField is null, while the relationship is not defined in the database.
```

Comment:
which relationship? the fkdefinition for source to target or between linkingobject::source and linkingobject::target?